### ISSUE TITLE
19/20 special sem 1 question 3

### ISSUE THREAD

**Issue by jordanwwb** [nus-cs2030/2021-s2#368]
hi, for this question the error that surface was name clash, they have the same method name and parameter but have different type erasure. Im not sure what mechanism is causing this error. 

![image](https://user-images.githubusercontent.com/77624953/116086031-f98bda80-a6d1-11eb-8c89-500b1de71d2d.png)
![image](https://user-images.githubusercontent.com/77624953/116086285-3bb51c00-a6d2-11eb-8f24-233af161f7c3.png)

**Comment by gracefour11** [comment]
this is related to type erasure i think... which won't be covered in finals

**Comment by Pchrome** [comment]
Yeah both will be 'erased' to just Stream type so its going to be processed as duplicated methods.

**Comment by gableejh** [comment]
seems like it's type erasure which will not be tested for finals

**Comment by JWulaXia** [comment]
type erasure, not covered, relax
